src/renderer/components/layout/sidebar-item.tsx

```diff
@@ -53,11 +52,9 @@ export class SidebarItem extends React.Component<SidebarItemProps> {
   }
 
   @computed get isExpandable(): boolean {
-    if (this.compact) {
-      return false; // not available currently
-    }
+    if (this.compact) return false; // not available in compact-mode currently
 
-    return Boolean(this.props.subMenus || this.props.children);
+    return Boolean(this.props.children);
```

Comment:
how about:

```ts
    return !this.compact || Boolean(this.props.children);
```

Reply:
I would like keep it as is for reasons:
1. current code imho more readable / intention clear
2. we could (and should) support it for `compact`-mode, so the line will be just removed.

Reply:
okay

Reply:
Though why did you format it at all then?